docker-compose.yml

```diff
@@ -8,7 +8,7 @@ services:
       - ganache
     environment:
       ethereum: http://ganache:8545
-      test_mnemonic: behave pipe turkey animal voyage dial relief menu blush match jeans general
+      test_mnemonic: myth like bonus scare over problem client lizard pioneer submit female collect
```

Comment:
just for curiosity ...why this was changed?

Reply:
I think it's because the new one is the default mnemonic of Ganache.

Reply:
This is the mnemonic that gnereated the accoutns of ganache when it runs in `--deterministic` mode but without inventing our own test mnemonic. https://github.com/daostack/devops#ganache

Reply:
(see https://github.com/daostack/subgraph/pull/32/files#diff-4e5e90c6228fd48698d074241c2ba760L50)